modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryLongestEpisodeView.kt

```diff
@@ -0,0 +1,82 @@
+package au.com.shiftyjelly.pocketcasts.endofyear.views.stories
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.compose.components.PodcastImage
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP50
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.endofyear.stories.StoryLongestEpisode
+import au.com.shiftyjelly.pocketcasts.models.db.helper.LongestEpisode
+import au.com.shiftyjelly.pocketcasts.settings.stats.StatsHelper
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+
+@Composable
+fun StoryLongestEpisodeView(
+    story: StoryLongestEpisode,
+    modifier: Modifier = Modifier,
+) {
+    val context = LocalContext.current
+    val timeText = remember {
+        StatsHelper.secondsToFriendlyString(
+            story.longestEpisode.duration.toLong(),
+            context.resources
+        )
+    }
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        PodcastImage(
+            uuid = story.longestEpisode.podcastUuid,
+            modifier = modifier
+                .size(200.dp)
+                .padding(top = 4.dp, end = 12.dp, bottom = 4.dp)
+        )
+        TextH30(
+            text = "The longest episode you listened to was ${story.longestEpisode.title} from the podcast ${story.longestEpisode.podcastTitle}",
```

Comment:
Can this be switched to a localised string?

Reply:
There are all placeholder strings. They'll be localized when I'll fine-tune design in future PRs.